### ISSUE TITLE
Potential merge - 2 terms for detection of stim involved in cytokinesis checkpoint?

### ISSUE THREAD

**Issue by dosumis** [geneontology/go-ontology#12326]
Are these two terms distinct?

**detection of stimulus involved in cytokinesis after mitosis checkpoint:** "Any detection of stimulus that is involved in cytokinesis after mitosis checkpoint."^^xsd:string
*EquivalentTo:* 'detection of stimulus' *that* **'part of'** *some* 'cytokinesis checkpoint'

**detection of stimulus involved in cytokinesis checkpoint:** "The series of events in which information about the formation and integrity of cytokinetic structures, such as the contractile ring, is received and converted into a molecular signal, contributing to a cytokinesis checkpoint."

Asking because the Equivalent Class definition of the first term is the obvious definition to use for the second + cytokinesis checkpoint is classified as a 'mitotic cell cycle checkpoint'.   If this classification is wrong, then the following solution might work:

**detection of stimulus involved in cytokinesis after mitosis checkpoint:** 
*EquivalentTo:* 'detection of stimulus' *that* **'part of'** *some* 'cytokinesis checkpoint' *and* **'part of'** *some* 'mitotic cell division' 

CC @ValWood

**Comment by ValWood** [comment]
I'm not sure what happened here.
The "cytokinesis after mitosis' checkpoint term was obsoleted, so all of the terms related to this should have done too?
http://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0000078

There are 2 checkpoints related to cytokinesis:

1. mitotic cell cycle checkpoint inhibiting CAR assembly
A Mad2-dependent mitotic cell cycle checkpoint which delays cytokinetic actinomycin ring assembly if there is a delay in early M-phase. 

2. cytokinesis checkpoint
A mitotic cell cycle checkpoint that detects a defect in cytokinesis and negatively regulates mitosis in the following cell cycle.

Any preexisting terms should have been obsoleted because their meanings were conflated.

**Comment by dosumis** [comment]
OK.  So solution is to either merge or obsolete - with obsolete being the safer option.

Judging from formerly entailed classifications I'm working through, There may other zombie terms like this under 'detection of stimulus involved in cell cycle checkpoint'.  Can you take a look?

One candidate:
GO:0072466	detection of stimulus involved in cell shape checkpoint

(No cell shape checkpoint term to point to)

**Comment by dosumis** [comment]
One quick question: aren't there equivalent checkpoints in meiosis?

**Comment by ValWood** [comment]
I would obsolete. I don't think any of these would have been used, but the terms were so unclear its a much safer option.

Same for cell shape checkpoint.

I think these were the only 2 which were removed.

**Comment by ValWood** [comment]
> One quick question: aren't there equivalent checkpoints in meiosis?

we don't know....there will be some checkpoints but I haven't seen them described.  We don't even use the equivalent cytokinesis terms for meiosis for yeast because  its "sporulation" so I'm a bit vague in this area.

**Comment by ukemi** [comment]
There is certainly cytokinesis in meiosis in animal cells. I think the original intent was to separate the mitotic/meiotic from the generic as was done in many other areas of this part of the ontology. We never expanded the meiotic terms fully. David, I'm going to assign this to you since it seems that you have been looking into these kinds of terms. Please let me know if this isn't appropriate.
-D

**Comment by ValWood** [comment]
I don't think we should add the specific meiotic cytokinesis checkpoint terms until we need them though....things might be different...

**Comment by dosumis** [comment]
The question is not whether there is cytokinesis in meiosis, but whether there is a checkpoint for it. 

Seems we should have some meiosis checkpoints during meiosis: https://en.wikipedia.org/wiki/Meiotic_recombination_checkpoint
This looks like a mini project in its own right,

Haven't found anything yet for cytokinesis, but haven't looked into it much. These just came up while trying to clean up un-entailed inferences.  Happy to do the obsoletions.  Reluctant to commit to new mini-project.

> I don't think we should add the specific meiotic cytokinesis checkpoint terms until we need them though....things might be different...

Probably are - but we often have general classes where there are differences in mechanism in more specific cases.  Can be useful as a way of finding similarities based on annotation.

**Comment by ukemi** [comment]
No need to commit to the mini-project. We knew it would be a project all on its own and consciously decided it was too big to handle in the initial cell cycle project. We just need to be sure that we leave things open enough that we don't have to do a massive overhaul of the existing structure when we decide to address the meiotic cell cycle.

**Comment by ValWood** [comment]
I would just rename the  existing term "cytokinesis checkpoint"
 as 'mitotic cytokinesis checkpoint'
it is defined and positioned mitotically. 

There is not much published on this yet. The existing experimental annotations (5 genes) are all from PomBase, and are all mitotic.

Easy to request terms for meiosis  using TG when we need them to annotate

**Comment by ValWood** [comment]
and obsolete the zombi terms...

**Comment by dosumis** [comment]
#12333

**Comment by dosumis** [comment]
@Ukemi - reassigning to you following [discussion of meiotic checkpoints in editor's meeting](http://gocwiki.geneontology.org/index.php/Ontology_meeting_2016-03-10#Cell_Cycle_Meiotic_versus_Mitotic), arising from #12333

Seems sensible to consider these issues together.

**Comment by ukemi** [comment]
Hi Val,

I changed the name as you suggested based on the definition and placement in the graph. 
Are these the 'zombie' terms?:
detection of stimulus involved in cytokinesis after mitosis checkpoint
signal transduction involved in cytokinesis after mitosis checkpoint

Neither one has any annotations. Based on the placement in the graph and the logical definitions, I think we should merge:
detection of stimulus involved in cytokinesis after mitosis checkpoint AND detection of stimulus involved in cytokinesis checkpoint

signal transduction involved in cytokinesis after mitosis checkpoint AND signal transduction involved in cytokinesis checkpoint

**Comment by ValWood** [comment]
I would keep the ones for cytokinesis checkpint
GO:0044878	cytokinesis checkpoint

and obsolete anything related to
GO:0000078 obsolete cytokinesis after mitosis checkpoint

we never really knew exactly what the second one meant... no point in having it hang around as a synonym

**Comment by ukemi** [comment]
The plot thickens...... This is a bit of a mess.

There is already a term GO:1990607, 'obsolete detection of stimulus involved in cytokinesis after mitosis checkpoint'

I suspect based on the logical definition and placement of GO:1903819 'detection of stimulus involved in cytokinesis after mitosis checkpoint' that this term was meant to be 'detection of stimulus involved in mitotic cytokinesis checkpoint'. I believe it does not really point to the obsolete term because the name of GO:0044878 used to be 'cytokinesis after mitosis checkpoint'. I think the term was never updated when GO:0044878 was updated. I propose we simply change the name and textual definition to update it in the same way that GO:0044878 has been updated.

GO:1903819 'detection of stimulus involved in cytokinesis after mitosis checkpoint' does not have an obsolete equivalent. Based on the logical definition and placement of this term, I suspect it was meant to be 'detection of stimulus involved in mitotic cytokinesis checkpoint', despite it's textual definition appearing to point to the obsolete term. I believe it does not really point to the obsolete term because the name of GO:0044878 used to be 'cytokinesis after mitosis checkpoint'. I think the term was never updated when GO:0044878 was updated.. I propose we simply change the name and textual definition to update it in the same way that GO:0044878 has been updated.

There are also these two terms:
GO:0072397 detection of stimulus involved in cytokinesis checkpoint
GO:0072398 signal transduction involved in cytokinesis checkpoint

These terms are children of the mitotic cell cycle checkpoints, but based on their definitions they are not mitosis-specific. I propose we move them to be children of the generic cell cycle checkpoint terms. If we want, we can create a term 'cytokinesis checkpoint' that is not mitosis-specific and also make these children of that new term. If we find that there is a spindle checkpoint in meiosis, then we can add the appropriate terms later. If we convince ourselves that there is no corresponding process in meiosis, then we can merge. I think this was the original plan, to leave the structure open enough to account for the meiosis terms if we needed them at a later date.

**Comment by ukemi** [comment]
As we discussed this morning. I have updated GO:1903819 and GO:1903820 to reflect their placement in the graph and their logical definitions. Both of those terms are now consistent with their parent. 
I have moved the generic terms out from under the mitosis node and we will either expand on them or merge them when we deal with meiosis. If there are no meiotic counterparts, then we will merge. If there are meiotic counterparts, then we will expand the graph to include meiosis-specific terms. 

Thanks to the Pombase curators for your help on this! It was nice to 'see' you all and be allowed to sit in on your group meeting for  a bit.

-D
